bundle/manifests/node-healthcheck-operator-manager-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  labels:
+    rbac.ext-remediation/aggregate-to-ext-remediation: "true"
```

Comment:
I remember @mshitrit said something about using an empty ClusterRole since all of the rules will get deleted

Reply:
are you talking about the aggregator?

Reply:
I don't remember the details. Only that adding the aggregated label on a cluster role could lead to wiping any other rules. @mshitrit knows better

Reply:
I think you mean that the aggregator cluster role should have empty rules declaration, which is already set here. I also deployed it and the aggregator works - once poison pill is deployed, all the rules are aggregated.